packages/helm-charts/celo-fullnode/templates/service.yaml

```diff
@@ -0,0 +1,22 @@
+{{ range $index, $e := until (int .Values.replicaCount ) }}
```

Comment:
I'm not too familiar with helm charts. Are these coded up by hand or do you use a tool for building these?

Reply:
It is possible to create a chart skeleton with `helm create <chart_name>`, but it will require in most cases a lot of changes. Helm charts are in essence the kubernetes yamls as templates, using Go templates and the `sprig` go template library. Also I find very useful checking existing charts (in this repository or publics in https://hub.helm.sh/) to see examples to different template approaches.